src/merklize/documentLoaders/dlJSONLD.ts

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+import { RemoteDocument, Url } from 'jsonld/jsonld-spec';
+
+import https from 'https';
+import http from 'http';
+import { parseLinkHeader } from 'jsonld/lib/util';
+import { LINK_HEADER_CONTEXT } from 'jsonld/lib/constants';
+import JsonLdError from 'jsonld/lib/JsonLdError';
+import { prependBase } from 'jsonld/lib/url';
+// eslint-disable-next-line  @typescript-eslint/no-var-requires
+const { httpClient } = require('@digitalbazaar/http-client');
```

Comment:
Can this pacckage be replaced to cross-fetch/polyfill or axios?

Reply:
Yes, I can replace the package. But go to implement it in a separate library.